src/mqt/bench/devices/rigetti.py

```diff
@@ -94,17 +95,23 @@ def __from_rigetti_index(cls, rigetti_index: int) -> int:
         row = rigetti_index // 100
         column = (rigetti_index % 100) // 10
         ring = rigetti_index % 10
-        return row * (ring_size * columns) + column * ring_size + ring
+        qubit_indx = row * (ring_size * columns) + column * ring_size + ring
+        if is_M3 and qubit_indx >= 70:
```

Comment:
Where does this number 70 come from? Can you maybe a small comment such that others understand why we do this.